strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/ActuatorEndpointControllerTest.java

```diff
@@ -73,4 +79,284 @@ public void testStrongboxInfo()
                .statusCode(HttpStatus.OK.value())
                .body("strongbox.revision", equalTo(revision));
     }
+
+    @Test
+    public void testMonitoringEndpointWithAuthorizedUser()
```

Comment:
we probably can avoid copy-pasting below test methods with `@ParameterizedTest`

Reply:
@fuss86 would you please check if this is possible? Thanks! :)

Reply:
yeah, done